recipes-devtools/rust/rust.inc

```diff
@@ -5,10 +5,7 @@ LICENSE = "MIT | Apache-2.0"
 LIC_FILES_CHKSUM ="file://COPYRIGHT;md5=43e1f1fb9c0ee3af66693d8c4fecafa8"
 
 inherit rust
-
-SRC_URI = "\
-	https://static.rust-lang.org/dist/${RUST_SNAPSHOT}.tar.gz;name=rust-snapshot \
```

Comment:
If I remember correctly I wanted to keep URLs for cargo snapshots, rust snapshots and rust-std snapshots in the same place and since cargos snapshot urls keep changing they have to be in different places. I can't remember if the rust snapshots also moved around.

If we remove 1.15.1 and if rust upstream decide not to change the URLs in the future there should be no problem putting it in rust.inc